lib/job/compute-next-run-at.ts

```diff
@@ -35,7 +35,7 @@ export const computeNextRunAt = function(this: Job) {
   const computeFromInterval = () => {
     debug('[%s:%s] computing next run via interval [%s]', this.attrs.name, this.attrs._id, interval);
     const dateNow = new Date();
-    let lastRun: Date = this.attrs.lastRunAt || dateNow;
+    let lastRun: Date = this.attrs.lastRunAt ?? dateNow;
     let { startDate, endDate, skipDays } = this.attrs;
     lastRun = dateForTimezone(lastRun).toDate();
     const cronOptions: any = { currentDate: lastRun };
```

Comment:
`const cronOptions = { currentDate: lastRun, tz: timezone };` wouuld infer the correct type all time